en/hardware/board_support_guide.md

```diff
@@ -19,21 +19,21 @@ The general requirements for all supported boards are:
    Board needs to pass acceptance criteria to ensure quality of parts and assembly.
 1. A clear and easy way to contact customer support for customers.
    One or more of the following is accepted:
-   - Slack channel presence
+   - PX4 Discord server presence
    - Support email
    - Phone number
 
-1. Point of contact (PoC) for the PX4 maintainers (direct email or available in Slack/Forum/Github)
-1. The board needs to use the [PX4 bootloader protocol](https://github.com/PX4/PX4-Autopilot/tree/main/platforms/nuttx/src/bootloader).
+2. Point of contact (PoC) for the PX4 maintainers (direct email or available in Slack/Forum/Github)
```

Comment:
markdown auto-numbers items if you use `1. `. This is better as an editor because you can move items without having to renumber.